motioneye/extra/linux_init

```diff
@@ -59,13 +59,19 @@ else
 fi
 
 # Pre-create config and data dirs and install configuration files
-[[ -d '/etc/motioneye' ]] || mkdir /etc/motioneye
-[[ -f '/etc/motioneye/motioneye.conf' ]] || cp extra/motioneye.conf.sample /etc/motioneye/motioneye.conf
+if [[ -f '/etc/motioneye/motioneye.conf' ]]
+then
+  # Update PID file directory if the default from old motionEye is still present: https://github.com/motioneye-project/motioneye/issues/2657
+  grep -q '^run_path /var/run$' /etc/motioneye/motioneye.conf && sed -i '/^run_path /var/run$/c\run_path /run/motioneye' /etc/motioneye/motioneye.conf
+else
+  [[ -d '/etc/motioneye' ]] || mkdir /etc/motioneye
+  cp extra/motioneye.conf.sample /etc/motioneye/motioneye.conf
```

Comment:
I don't know how often people make manual changes to `motioneye.conf`, but I think that is at least a possibility that shouldn't be disregarded. With these changes, any manual changes would be lost and unless the admin happens to be aware of init overwriting the file it might cause confusion and frustration. I do see the point in overwriting the file, but should we at least signal the user that this happened and make a backup and/or show a diff or something that helps in both noticing what happened and re-doing the changes that are still needed?

Reply:
If you mean the `cp`, note that this is only done if the file does not exist, so nothing is ever overwritten aside of the `run_path` setting, but only if it is exactly the old default value from first to last character, which cannot work at all with the new systemd unit and also uses the deprecated `/var/run` which can in general only work as/if it is a symlink to `/run` for backwards-compatibility reasons.

Reply:
Oh, my bad for losing the context after writing the first comment. This is absolutely fine.